### PR TITLE
Add CI jobs for libvirt-s390x

### PR DESCRIPTION
**Pull request by deepsm007**

### CODE REVIEW

File: core-services/template-deprecation/_allowlist.yaml [hunk 1]
```diff
@@ -1888,6 +1888,9 @@ templates:
           release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.6:
             generated: false
             kind: release
+          release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.7:
+            generated: false
+            kind: release
```
Comment (jaypoulz) [inline core-services/template-deprecation/_allowlist.yaml:1893]: You have a white space issue on this line :)
Check if you have spaces / tabs after release

**Comment by jaypoulz** [comment]
The yaml linter also has fixes for you :)

**Comment by jaypoulz** [comment]
Also, can you add the cluster sample job configs to this patch as well? That should be a copy+paste over.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 2]
```diff
@@ -617,6 +617,211 @@ periodics:
...
+            "[sig-network][Feature:Router] The HAProxy router should serve a route that points to two services and respect weights [Suite:openshift/conformance/parallel]"
+            "[sig-network][Feature:Router] The HAProxy router should serve routes that were created from an ingress [Suite:openshift/conformance/parallel]"
+            "[sig-network][Feature:Router] The HAProxy router should serve the correct routes when scoped to a single namespace and label set [Suite:openshift/conformance/parallel]"
+            "[sig-operator] an end user can use OLM can subscribe to the operator [Suite:openshift/conformance/parallel]"
+            "[sig-storage] CSI mock volume CSI Volume expansion should expand volume by restarting pod if attach=on, nodeExpansion=on [Suite:openshift/conformance/parallel] [Suite:k8s]"
+            "[sig-storage] Dynamic Provisioning [k8s.io] GlusterDynamicProvisioner should create and delete persistent volumes [fast] [Skipped:ibmcloud] [Suite:openshift/conformance/parallel] [Suite:k8s]"
+           EOEXCLUDE
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:770]: Make sure this line is properly lined up, I'm seeing `/bin/bash: line 193: warning: here-document at line 141 delimited by end-of-file (wanted `EOEXCLUDE')`

Comment (deepsm007) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml]: Fixed

**Comment by jaypoulz** [comment]
Ah the irony of jenkins & image-eco swapping on the rebuild :D

**Review by droslean — APPROVED** [review]

**Pull request merged**